dapr/proto/common/v1/common.proto

```diff
@@ -40,7 +40,7 @@ message HTTPExtension {
   Verb verb = 1;
 
   // querystring includes HTTP querystring.
-  map<string, string> querystring = 2;
```

Comment:
Worth documenting the assumptions here (or elsewhere). 

It looks like the query string is assumed to NOT include the leading `?`.

It looks like the query string is assumed to be in encoded form (based on `EncodeHTTPQueryString` returning it verbatim). Are these the same assumptions we have about `InvokeRequest.method`?

Reply:
Added comment.